src/lib/charts/PieChart/PieChart.tsx

```diff
@@ -0,0 +1,36 @@
+import { ChartColor, ChartOptions } from 'chart.js';
+import { merge } from 'lodash';
+import React from 'react';
+import { Pie, ChartData } from 'react-chartjs-2';
+
+export const PieChart: React.FC<{
+  data?: {
+    label: string;
+    value: number;
+    color?: ChartColor;
+  }[];
+  chartJSData?: ChartData<Chart.ChartData>;
+  options?: ChartOptions;
+}> = props => {
```

Comment:
We should always separate out the prop types into a separate export so they may be imported and extended as needed in any project using our component library.

Reply:
I went ahead and took care of this.